.github/workflows/install.yml

```diff
@@ -0,0 +1,52 @@
+name: "Install"
```

Comment:
Could we put these 2 jobs into https://github.com/python-gitlab/python-gitlab/blob/main/.github/workflows/test.yml?

We define some additional workflow config there, so this way we make sure the changes apply to the jobs consistetly.